machines/QrLoginMachine.ts

```diff
@@ -483,6 +523,11 @@ export const qrLoginMachine =
         },
       },
       guards: {
+        isConsentGiven: context => {
+          console.log('isConsentGiven value', context.showFaceAuthConsent);
+          return !context.showFaceAuthConsent;
```

Comment:
guard says `isConsentGiven` -> so expectation would be, we will get true if consent is given. So don't use negation